src/App.tsx

```diff
@@ -1,26 +1,290 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { USER_ID, deleteTodo, getTodos, postTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import cn from 'classnames';
+import { TodoItem } from './components/TodoItem';
 
-const USER_ID = 0;
+type Sort = 'All' | 'Active' | 'Completed';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortBy, setSortBy] = useState<Sort>('All');
+  const [title, setTitle] = useState('');
+  const [loadingIds, setLoadingIds] = useState<number[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [tempState, setTempState] = useState<Todo | null>(null);
+
+  const inputRef = useRef<HTMLInputElement | null>(null);
+
+  useEffect(() => {
+    inputRef.current?.focus();
+
+    getTodos()
+      .then(todosFromServer => {
+        setTodos(todosFromServer);
+      })
+      .catch(() => setErrorMessage('Unable to load todos'));
```

Comment:
All messages need to move in enums ( in a separe file). Comment for all files and places in your app.